### PR TITLE
Fix error in export to Ocellus

### PR DESCRIPTION
**Pull request by wooyey**
Some fight between review comments, linter and my understanding of JS.

### CODE REVIEW

File: cvat-core/src/api-implementation.js [hunk 1]
```diff
@@ -415,17 +415,21 @@
 
         cvat.aifred.getWorkoutMetrics.implementation = (workoutId) => aifredProxy.getWorkoutMetrics(workoutId);
 
-        cvat.aifred.getOcellusModel.implementation = (workoutId, fileName) => {
-            aifredProxy.getOcellusModel(workoutId, fileName);
-        };
+        cvat.aifred.getOcellusModel.implementation =
+            (workoutId, fileName) => aifredProxy.getOcellusModel(workoutId, fileName);
```
Comment (kcarlson) [inline cvat-core/src/api-implementation.js:419]: Maybe this? Or is it a python convention?
```python
cvat.aifred.getOcellusModel.implementation = (workoutId, fileName) => 
    aifredProxy.getOcellusModel(workoutId, fileName);
```

**Review by kcarlson — APPROVED** [review]
lgtm, just a formatting question

File: cvat-core/src/api-implementation.js (hunk 1 above)
Comment (wooyey) [reply cvat-core/src/api-implementation.js:419]: Nope Senior you've just thrown me on a mine field called "JS linter":
```
Expected no linebreak before this expression.eslintimplicit-arrow-linebreak
```
So I cannot use break line after `=>` in this project ;)

Comment (kcarlson) [reply cvat-core/src/api-implementation.js:419]: wat!? I'd say the linter is wrong :P

**Pull request merged**